scripts/core/ProcessManager.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+#define _CRT_SECURE_NO_WARNINGS
+#include "Config.h"
+#include <Windows.h>
+#include <string>
+#include <vector>
+#include <unordered_set>
+#include <winternl.h>
+
+typedef NTSTATUS(NTAPI* pfnNtSuspendProcess)(IN HANDLE ProcessHandle);
+typedef NTSTATUS(NTAPI* pfnNtResumeProcess)(IN HANDLE ProcessHandle);
+
+class ProcessManager {
+public:
+    ProcessManager(const ConfigData& config);
+
+    void main_loop();
+
+private:
+    enum class State {
+        IDLE,
+        BOOSTING
```

Comment:
Active would sound better here
```suggestion
        ACTIVE
```